lib/tasks/election.rake

```diff
@@ -27,4 +27,22 @@ namespace :election do
       )
     end
   end
+
+  task :end_ministerial_appointments, [:end_date] => :environment do |_t, args|
+    appointments = RoleAppointment.current.for_ministerial_roles
+    end_date = Date.today
+    prime_ministerial_role_id = 1
+    begin
+      if args[:end_date]
+        end_date = Date.parse(args[:end_date])
```

Comment:
Yeah.. that would be the format. The intention is that we won't need to provide a date and `today` will be fine but as we don't want to be deploying tomorrow or over the weekend I wanted to give it a bit more flexibility just in case.

I've added some documentation in b1a854f